### ISSUE TITLE
My automatic install stopped working

### ISSUE THREAD

**Issue by w2ak** [go-gitea/gitea#4229]
- Gitea version (or commit ref): docker hub's gitea:latest
- Operating system: docker in Ubuntu 16.04
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant

## Description

So I've been using ansible to automatically install *and* configure Gitea. I build my own docker image from gitea and have an entrypoint that creates the admin if necessary.

As you can see in my [docker-compose](https://paste.neze.fr/iruxopemam.j2) file, the `INSTALL_LOCK` variable is always set to true.

**However, I end up on the Gitea install web page.**

As you can see in my [Dockerfile](https://paste.neze.fr/jidofekibe), I only append [app_custom.ini](https://paste.neze.fr/ukifutohuz.j2) to the original default template. This was working before (around may 30 according to my git log).

**However, after starting the container, the configuration file only contains the INTERNAL_KEY.**

Finally, here is my [docker-entrypoint.sh](https://paste.neze.fr/bepuyivewo.bash). I haven't changed it since the time it worked. When it worked, it would show "Generating secret key" then start the server. Then when my `CREATE_ADMIN` variable was set in `docker-compose.yml` it would show in parallel "Creating admin user".

**However, now I see only "Generating secret key" then your `/usr/bin/entrypoint` serves.** It appears like it ignores every environment variable, and my admin generation code never runs.

I do not have any idea about where it comes from :worried:
Thank you by advance.

**Comment by lunny** [comment]
What's your latest work instance gitea version?

**Comment by w2ak** [comment]
If you're asking what is the latest with which it was working, I have been trying to find it but I cannot manage to find it. The problem might come from outside of gitea (docker-compose, the s6 image, etc.), I am just not experienced enough to actually have an idea which is why I am asking the question.

If you're asking what is the latest version I tested, I am testing right now.

WUT! So today's version is 6efdcae, and I don't understand my install worked...

I have another issue but I will file it separately.

Do you have any idea what could be causing this change? I am really surprised...
